packages/jest-cli/src/printTypeahead.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {Config, Path} from 'types/Config';
+
+const {relativePath} = require('./reporters/utils');
+const ansiEscapes = require('ansi-escapes');
+const chalk = require('chalk');
+const highlight = require('./lib/highlight');
+const path = require('path');
+
+const pluralizeFile = (total: number) => total === 1 ? 'file' : 'files';
+
+const printTypeahead = (
+  config: Config,
+  pipe: stream$Writable | tty$WriteStream,
+  pattern: string,
+  allResults: Array<Path>,
+  max: number = 10
+) => {
+  const total = allResults.length;
+  const results = allResults.slice(0, max);
+
+  pipe.write(ansiEscapes.eraseDown);
+  pipe.write(`${chalk.dim(' pattern \u203A')} ${pattern}`);
+  pipe.write(ansiEscapes.cursorSavePosition);
+
+  if (pattern) {
+    if (total) {
+      pipe.write(`\n\n Pattern matches ${total} ${pluralizeFile(total)}...`);
```

Comment:
Why the `...` here? IMO it's unnecessary.

Usually `...` is used in the UI on the buttons that present an additional dialog (e.g. `Open...`).
In this case just `.` will do.